pages/advice/index.js

```diff
@@ -3,13 +3,16 @@ import { useState } from "react";
 import Footer from "../../components/Footer";
 import RandomButton from "../../components/RandomImageButton";
 import RandomAdvice from "../../components/RandomAdvice";
+import { generateRandomString } from "../../helpers/generateRandomString";
 
 export default function Advice() {
   const [advice, setAdvice] = useState("");
+  const url =
+    "https://api.adviceslip.com/advice?random=" + generateRandomString(12);
 
   async function fetchAdvice() {
     try {
-      const response = await fetch("https://api.adviceslip.com/advice");
+      const response = await fetch(url);
       const data = await response.json();
       setAdvice(data.slip);
```

Comment:
What does data.slip do? I tried googling but I couldn't find anything about it...